WhatIsASprint.md

```diff
@@ -0,0 +1,23 @@
+
+# **What is a Sprint?**
+
+
+A **_sprint_** is a set period of time broken out from a project's overall timeline with associated tasks to be completed during that period of time. That list of tasks is referred to as the sprint's <b>Product Backlog</b>.
```

Comment:
This is my first time using markdown and was attempting to both bold and add italics. As for the second comment, I am aware of what markdown does but thank you for the explanation.